src/libraries/System.Text.Json/tests/System.Text.Json.FSharp.Tests/EnumTests.fs

```diff
@@ -15,6 +15,17 @@ type BadEnum =
 let badEnum = BadEnum.``There's a comma, in my name`` ||| BadEnum.``There's a comma, even here``
 let badEnumJsonStr = $"\"{badEnum}\""
 
+type NumericLabelEnum =
+  | ``1`` = 1
+  | ``2`` = 2
+  | ``3`` = 4
+  
+let enumWithNumericValue = NumericLabelEnum.``1``
+let enumWithNumericValueJsonStr = $"\"{enumWithNumericValue}\""
```

Comment:
Probably better to just express the expected value as a literal string.

Reply:
Fixed.